tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java

```diff
@@ -269,6 +269,42 @@ public void testStartWithParms() {
         // Close the terminal.
         terminal.close();
     }
+    
+    /**
+     * Tests the start with parameters command string for proper composition.
```

Comment:
I don't think this test is worth adding.  

Maybe it was a bad idea.  

To at least illustrate roughly what I was thinking, consider this.
https://github.com/scottkurz/liberty-tools-eclipse/tree/start-with-params

Note though we have to refactor the bundle by exposing / consuming this util package to test in this fashion.

Let's talk more on our scrum.

Reply:
this re-working of the order of user parameters does not work with gradle.  if, say, '--hotTests' is placed before the LibertyDev command, gradle fails with an unknown cmd arg failure. Gradle thinks the --hotTests arg applies to gradle itself and not th eLibertyDev call.

Reply:
Let's forget this idea then.  I hadn't thought this through.  We'll go with the existing behavior, update the UFO, and revisit this in:  https://github.com/OpenLiberty/liberty-tools-eclipse/issues/111   for MVP.